app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -22,12 +22,12 @@ def linkedin
     if @user && @user.persisted?
       sign_in @user, event: :authentication #this will throw if @user is not activated
       set_flash_message(:notice, :success, kind: "LinkedIn") if is_navigational_format?
-      redirect_to employer_home_path
+      redirect_to edit_employer_path(current_user.employer)
```

Comment:
Bouncing straight to the edit employer page makes sense on first login but less so after that.